commands/utility/twitch/twitchApi.py

```
@@ -37,9 +38,16 @@ async def get_app_access_token(self) -> None:
             "grant_type": "client_credentials",
         }
 
-        async with self.session.post(url=auth_url, params=params) as response:
-            data = await response.json()
-            self.access_token = data["access_token"]
+        try:
+            async with self.session.post(url=auth_url, params=params, timeout=ClientTimeout(total=10)) as response:
+                data = await response.json()
+                self.access_token = data["access_token"]
+        except (asyncio.TimeoutError, aiohttp.ClientError) as e:
```

Flake8 (7.3.0)

[error] undefined name 'asyncio'

(F821)